Sources/POSIX/Error.swift

```diff
@@ -38,7 +38,7 @@ import func libc.strerror_r
 extension SystemError: CustomStringConvertible {
     public var description: String {
         func strerror(_ errno: Int32) -> String {
-            for cap in [64, 128, 512, 1024, 2048, 4096, 8192, 16_384] {
+            for cap in sequence(first: 64, next: { $0 < 16 * 1024 ? $0 * 2 : nil }) {
```

Comment:
I considered this, but I think it is actually less readable...

IMHO, the first version was actually best it just needed an extra condition to break out once past the max size.